### ISSUE TITLE
Create tests for the case of #303

### ISSUE THREAD

**Issue by ankurankan** [pgmpy/pgmpy#305]

**Comment by sharangdhar** [comment]
Has anyone worked on this yet? There seem to be some basic tests already in place.

**Comment by ankurankan** [comment]
@sharangdhar Yeah there are a few tests for `map_inference` already but there are a lot of edge cases for which you can write tests.

**Comment by ibrahim5253** [comment]
Is it still open?

**Comment by ankurankan** [comment]
@ibrahim5253 You can come up with some edge cases and add tests for that.

**Comment by shubha987** [comment]
@ankurankan can you assign this to me?  I want to give it a shot

**Comment by ankurankan** [comment]
@shubha987 I do not remember what exactly issue #303 was. I would suggest first checking if there are any tests missing for the `map_query` method, or if it has already been added.

**Comment by shubha987** [comment]
okay i will check first

**Comment by shostykovich21** [comment]
@ankurankan If this issue is still open, I'd like to work on it. I noticed a few test cases that might be missing, like when only variables are provided (no evidence) or vice versa. There may be a couple more as well... I can dig deeper and help expand the coverage.

**Comment by ankurankan** [comment]
I think the actual issue isn't clear anymore, so I will close this. @shostykovich21 Feel free to create another issue for improving the test coverage if you see that the current tests are missing any cases.
